web/docs/user-guides/install-a-package.md

```diff
@@ -0,0 +1,227 @@
+# Install a Package
+
+A package is a set of pipelines and contexts that are thematically related and
+deployed together as a single unit.
+
+## Anatomy of a Package
+
+A package definition consists of three major parts.
+
+1. Package metadata, for example the name, author and description of the
+   package.
+2. Snippets, which are example pipelines for use with the Explorer on
+   [app.tenzir.com](https://app.tenzir.com/explorer) that demonstrate how to use
+   the package.
+3. A set of inputs provided by the user to customize the package installation
+   to their environment.
+4. A set of pipelines and contexts that make up the contents of the package.
```

Comment:
The guide is called "Install a Package", but starts with something that is aimed at package creators. As someone who just wants to install a package, I have to skip all the way down to "Install from the Tenzir Library".

I think this should either be separate guides, or we should switch the order, but we'd need some restructuring in that case... Alternatively, we can make this change after the next release if that's more convenient.

Reply:
We discussed in a call that we're leaving this as-is for now, and add some restructuring when we add a separate "Write a package" guide.